### PR TITLE
Days of week option and windows setup

### PR DESCRIPTION
**Pull request by kArmadi110**

### CODE REVIEW

File: parkanizerer.cmd [hunk 1]
```diff
@@ -0,0 +1 @@
+python main.py
```
Comment (maraid) [inline parkanizerer.cmd:1]: Is this needed? With the  hasbang #!/usr/bin/env python3 the script should be runnable directly, but I'm not sure if it is accepted as startup script.

File: main.py [hunk 2]
```diff
@@ -28,15 +47,15 @@ def fetch_zone_id(zone_name):
     return next(zone["id"] for zone in r.json()["zones"] if zone["name"] == zone_name)
 
 
-def fetch_desk_id(desk_name, zone_id):
+def fetch_desk_id(desk_name, zone_id, days_to_take):
```
Comment (maraid) [inline main.py:50]: single day_to_take
```suggestion
def fetch_desk_id(desk_name, zone_id, day_to_take):
```

File: main.py [hunk 3]
```diff
@@ -20,6 +20,25 @@ def one_week_from_now():
...
+    for dow in config["days_of_week"]:
+        next_day = datetime.today()
+        
+        days_left = dow - next_day.weekday()
+        
+        if days_left <= 0:
+            days_left += 7
+            
+        next_day += timedelta(days=days_left)
+
```
Comment (maraid) [inline main.py:38]: This calculation is a bit all around. Can I suggest a denser form?

```suggestion
        today = datetime.today()
        days_left = (dow - today.weekday()) % 7
        next_day = today + datetime.timedelta(days=days_left)
```

File: main.py [hunk 4]
```diff
@@ -20,6 +20,25 @@ def one_week_from_now():
     next_week = datetime.today() + timedelta(days=7)
     return next_week.strftime(r"%Y-%m-%d")
 
+def get_next_week_days():
+    result = []
+    
+    if not config["days_of_week"]:
```
Comment (maraid) [inline main.py:26]: Please use config.get() for backwards compatibility
```suggestion
    if not config.get("days_of_week", []):
```

File: main.py [hunk 5]
```diff
@@ -20,6 +20,25 @@ def one_week_from_now():
...
+        result.append(one_week_from_now())
+
```
Comment (maraid) [inline main.py:28]: Please add else branch here for clarity

**Review by maraid — CHANGES_REQUESTED** [review]
Thanks for the PR, it's greatly appreciated. I added some comments

File: parkanizerer.cmd (hunk 1 above)
Comment (kArmadi110) [reply parkanizerer.cmd:1]: Don't quote me on that but afaik by default shebang isn't supported on windows. The other option would be to handle this by file extension association but for most developers the .py extension is associated with an IDE and not with the interpreter.
We can remove thus tough and instruct the user to create it alongside a shortcut.

File: main.py (hunk 3 above)
Comment (kArmadi110) [reply main.py:38]: Of course.

File: main.py (hunk 5 above)
Comment (kArmadi110) [reply main.py]: else branch added

**Review by maraid — APPROVED** [review]

**Pull request merged**